bin/agat_sp_manage_functional_annotation.pl

```diff
@@ -13,22 +13,26 @@
 use Bio::Tools::GFF;
 use AGAT::Omniscient;
 
+#use Data::Dumper; # JN: for dedug printing
+my $DEBUG = 1;    # JN: for dedug printing
```

Comment:
Should it be an option? otherwise has to be removed.
See the debug call later in the code

Reply:
Yes, this could definitely be removed for a final (possible) merge with master. For now, I kept it due to the `TODO` I unfortunately still have in the code: The part where I'm unsure what happens if we at some point may have good blast hits with different gene names. I still not 100% sure if this is a problem or not, that's why I kept the debug printing to be able to see what cases where this might apply. Setting the `$DEBUG = 0;` quickly disables this printing, so I kept it for now -- for you to comment!